bin/plotting/pycbc_plot_qscan

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2018 Ian Harry
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+Plot single-detector qscan of strain data
+"""
+
+import sys
+import argparse
+import logging
+import numpy
+
+import matplotlib
+matplotlib.use('Agg')
+from matplotlib import pyplot as plt
+from matplotlib.colors import LogNorm
+
+import pycbc.strain
+import pycbc.version
+import pycbc.results
+
+# https://stackoverflow.com/questions/9978880/python-argument-parser-list-of-list-or-tuple-of-tuples
+def t_window(s):
+    try:
+        start, end = map(float, s.split(','))
+        return [start, end]
+    except:
+        raise argparse.ArgumentTypeError("Input must be start,end start,end")
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("--version", action="version",
+                    version=pycbc.version.git_verbose_msg)
+parser.add_argument('--output-file', required=True, help='Output plot')
+parser.add_argument('--center-time', type=float,
+                    help='Center plot on the given GPS time')
```

Comment:
Is `--center-time` not a required argument?

Reply:
Nevermind, it makes sense after reading the other options.

Reply:
Added required=True here.